package.json

```diff
@@ -44,5 +49,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "relativeDependencies": {
```

Comment:
I have this here for local development, but if you don't have these packages locally at those paths, it'll still work! Will just pull the required version from npm instead